content/issues/2025-10-16-incrementals-outage.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Outage on incrementals.jenkins.io
+date: 2025-10-16T16:00:00-00:00
```

Comment:
```suggestion
date: 2025-10-15T15:41:00-00:00
```

(Just caught a few logs after you started the PR, proving the issue started yesterday)